build.gradle

```diff
@@ -150,10 +150,25 @@ idea {
     }
 }
 
+tasks.withType(AbstractArchiveTask).configureEach {
+    // Reproducible jar files
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+}
+
 task runtimeJar(type: Jar) {
     dependsOn compileJava
     archiveBaseName = 'rhino-runtime'
     from sourceSets.main.output
+    from ('LICENSE.txt') {
+        into 'META-INF'
+    }
+    from ('NOTICE.txt') {
+        into 'META-INF'
+    }
+    from ('NOTICE-tools.txt') {
```

Comment:
Think `NOTICE-tools.txt` doesn't need to be included in the runtime jar, as that one doesn't contain the tool source, no?

Reply:
True, fixed.